de/extend_your_application/README.md

```diff
@@ -148,25 +148,25 @@ Das sieht dann so aus:
 {% extends 'blog/base.html' %}
 
 {% block content %}
-    <div class="post">
-        {% if post.published_date %}
-            <div class="date">
-                {{ post.published_date }}
-            </div>
-        {% endif %}
-        <h1>{{ post.title }}</h1>
-        <p>{{ post.text|linebreaksbr }}</p>
-    </div>
+    <div class="post">
+        {% if post.published_date %}
+            <div class="date">
+                {{ post.published_date }}
+            </div>
+        {% endif %}
+        <h1>{{ post.title }}</h1>
+        <p>{{ post.text|linebreaksbr }}</p>
+    </div>
 {% endblock %}
 ```
 
 Wir erweitern wieder `base.html`. Im `content`-Block wollen wir das Publikationsdatum eines Posts (published_date), falls es existiert, anzeigen und auch den Titel und den Text. Aber wir müssen noch ein paar wichtige Dinge klären, oder?
 
-{% raw %} {% if ... %} ... {% endif %}</code> ist ein Template-Tag, das wir benutzen können, wenn wir etwas überprüfen möchten. (Erinnerst du dich? `if ...else</0> vom <strong>Einführung in Python</strong> Kapitel?) In diesem Fall hier wollen wir prüfen, ob das <code>published_date` Feld eines Post-Objektes nicht leer ist. {% endraw %}
+{% raw %}`{% if ... %} ... {% endif %}` ist ein Template-Tag, das wir benutzen können, wenn wir etwas überprüfen möchten. (Erinnerst du dich an `if ... else` vom Kapitel **Einführung in Python**?) In diesem Fall hier wollen wir prüfen, ob das `published_date</0> Feld eines Post-Objektes nicht leer ist. {% endraw %}</p>
 
-OK, aktualisieren wir unsere Seite und sehen, ob `TemplateDoesNotExist` jetzt weg ist.
+<p>OK, aktualisieren wir unsere Seite und sehen, ob <code>TemplateDoesNotExist` jetzt weg ist.
 
```

Comment:
@das-g, any idea whats up here with these tags? I am not sure how this works, the closing tag of 'p' is ealier (line 165) than the opening tag (167)?

Reply:
Probably a mistake in the previous "sentence": We had

    else</0> vom Kapitel <1>Einführung in Python</1>?) In diesem Fall hier wollen wir prüfen, ob das <2>published_date</0> Feld eines Post-Objektes nicht leer ist. {% endraw %}

instead of

    else</0> vom Kapitel <1>Einführung in Python</1>?) In diesem Fall hier wollen wir prüfen, ob das <2>published_date</2> Feld eines Post-Objektes nicht leer ist. {% endraw %}

(i.e. mismatched tag numbers in `<2>published_date</0>`)

I don't know whether the mismatched `</0>` at the beginning (carried over from the "sentence" yet before that, because Crowdin splits at the last `.` of `...`) is also a problem. If it is, we have to switch back to your approach of splitting the German translation differently than Crowdin splits the English original, so that opening and closing tags stay together.